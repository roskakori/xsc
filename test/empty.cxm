<?xml version="1.0" encoding="utf-8"?>
<!-- Various constructs left empty -->
<data>
  <empty/>
  <?cxm for customers?><?cxm end for?>
  <?cxm if True?><?cxm end if?>
</data>
