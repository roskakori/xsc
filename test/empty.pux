<?xml version="1.0" encoding="utf-8"?>
<!-- Various constructs left empty -->
<data>
  <empty/>
  <?pux for customers?><?pux end for?>
  <?pux if True?><?pux end if?>
</data>
