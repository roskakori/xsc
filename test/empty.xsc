<?xml version="1.0" encoding="utf-8"?>
<!-- Various constructs left empty -->
<data>
  <empty/>
  <?xsc for customers?><?xsc end for?>
  <?xsc if True?><?xsc end if?>
</data>
