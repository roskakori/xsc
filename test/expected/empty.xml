<?xml version="1.0" encoding="utf-8"?><!-- Various constructs left empty --><data>
  <empty/>
  
  
</data>